webpack.config.js

```diff
@@ -159,23 +160,22 @@ module.exports = env => {
         },
         {
           test: /\.(scss|css)$/,
-          use: ExtractTextPlugin.extract({
-            use: [
-              {
-                loader: 'css-loader',
-                options: {
-                  sourceMap: true,
-                },
+          use: [
+            MiniCssExtractPlugin.loader,
+            {
+              loader: 'css-loader',
+              options: {
+                sourceMap: true,
               },
-              {
-                loader: 'sass-loader',
-                options: {
-                  sourceMap: true,
-                  precision: 10, // fixes line height issue with bootstrap button addons
```

Comment:
Can you keep this comment around? Just cuz it's so random and I don't want someone accidentally deleting the precision config. Or if it's no longer needed, toss the config?

But seems like it's still needed. https://github.com/twbs/bootstrap-sass#sass-number-precision. You could add that link to give it more context. Might also be able to drop the precision from 10 to 8...